client/components/Profile.jsx

```diff
@@ -34,24 +38,51 @@ class Profile extends React.Component {
               <fieldset>
                 <legend>Profile</legend>
 
-                <label htmlFor='username'>Username</label>
+                {/* <img src={profilePic} alt={name}> </img> */}
+
+                <label htmlFor='username'>Username: </label>
                 <input id='username' name='username' placeholder='username'
                   onChange={this.handleChange} value={username} />
+                <br/>
+
+                <label htmlFor='email'>Email: </label>
+                <input id='email' name='email' placeholder='email'
+                  onChange={this.handleChange} value={email} />
+                <br/>
+
+                <label htmlFor='cohort'>Cohort: </label>
+                <input id='cohort' name='cohort' placeholder='cohort'
+                  onChange={this.handleChange} value={cohort} />
+                <br/>
```

Comment:
Cohort should only be shown on the grad profile because it isn't relevant for staff and employers. So we can remove it from here.

Reply:
This has been deleted in the latest changes (which we weren't able to push)